src/tests/ftest/recovery/pass_2.py

```diff
@@ -0,0 +1,211 @@
+"""
+  (C) Copyright 2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+from ClusterShell.NodeSet import NodeSet
```

Comment:
very minor - recommended to leave a space between "built-in" and 3rd-party imports
```suggestion
import time

from ClusterShell.NodeSet import NodeSet
```

Reply:
Done